### ISSUE TITLE
Bug: wrong title & aria-label for images

### ISSUE THREAD

**Issue by therobrob** [fipguide/fipguide.github.io#173]
Our Images use the file-name for title and aria-label attribute. Unfortunately, this is wrong.
They´re decorative images und should be hided (empty title & empty aria-label).
